api/google-manager/services/google-manager.js

```
@@ -110,8 +110,16 @@ class GoogleManager {
 
       return group.data;
     } catch(err) {
-      console.error(`Google Admin Directory API returned error ${err} when getting group`);
-      throw new Error(err);
+      if(err.code == 404) {
+        console.error('Google group does not exist');
+        /*
+          we have to return undefined to allow strapi's code
+          to work even if the project doesn't have a google group
+        */
+        return undefined;
```

Should we return a descriptive error?